src/Ui/Table/Component/Header/Header.php

```diff
@@ -100,15 +100,13 @@ public function setBuilder(TableBuilder $builder)
      */
     public function getDirection($default = null)
     {
-        $query = $_GET;
+        $builder = $this->getBuilder();     
```

Comment:
The good style is to put a type in the comment above
```php
/** QueryBuilder $builder */
$builder = $this->getBuilder();
```

Reply:
I've left this as-is, as I've not modified this line concerning the QueryBuilder.